ml-agents/mlagents/trainers/learn.py

```diff
@@ -49,6 +50,23 @@ def parse_command_line(argv: Optional[List[str]] = None) -> RunOptions:
     return RunOptions.from_argparse(args)
 
 
+def _Get_checkpoint_name(
```

Comment:
```suggestion
def _get_checkpoint_name(
```
snake_case_for_python_functions